src/components/dashboard/sidebar/DashboardSidebar.jsx

```diff
@@ -9,7 +9,7 @@ import SidebarCard from './SidebarCard';
 import { UserSubsidyContext } from '../../enterprise-user-subsidy';
 
 export const CATALOG_ACCESS_CARD_TITLE = 'Catalog Access';
-export const CATALOG_ACCESS_CARD_BUTTON_TEXT = 'Find a course';
+export const CATALOG_ACCESS_CARD_BUTTON_TEXT = 'Find a Course';
```

Comment:
Noticed this right after deploying; we use a capital C everywhere else on the dashboard 🙃